Assets/Scripts/Constants.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Constants : MonoBehaviour 
+{
+	private const string MAIN_MENU_SCENE_NAME = "MainMenu";
```

Comment:
@NatashaHMP,  Eu tenho uma boa e uma Má notícia. 

A boa é que Foi isso mesmo que combinamos, perfeito, você pegou a ideia e mandou bala essa é a boa noticia, a má noticia é que existe alguns probleminhas com essa classe, vou resumir aqui mas qualquer coisa te ligo na segunda feira e te explico.

O primeiro problema é com o Nome da classe, pois só constantes não define exatamente o que sua classe contem, minha sugestão é que vc crie uma classe para cada tipo de constante, exemplo, LevelConstants, TagsConstants etc.. Ou você pode agrupar os tipos dentro de uma classe chamada Constants. segue os exemplos ======

Existem algumas sugestões Exemplo 1: 

```
// Separaria as constants assim, e não precisa herdar de MonoBehaviour, pois a classe não será processada pela,
// framework da unity
public sealed class SceneNameConstants {
    public const string LEVEL_SELECTION = "SelectLvl";
    public const string THREE_VALLEY = "MainLevel";
    public const string MENU = "Menu";
    public const string PAUSE = "Pause";
    public const string OPTIONS = "Options";
}

public sealed class TagsConstants {
    public const string PLAYER = "Player";
    public const string CAR = "Car";
    public const string MOTO_CROSS = "MotoCross";
    public const string PLAYER_LIFE = "PlayerLife";
    public const string UI_SCORE = "UIScore";
}

//Exemplo de como voce utilizaria == O nome da classe abaixo e so exemplo 
public class MenuSceneController {
    
    private void Update() {
        GotoScene(SceneNameConstants.THREE_VALLEY);
        GetGameObjectWithTag(TagsConstants.PLAYER);
    }
    
    private void GotoScene( string sceneName ) {
        //TODO: Faz ir para outra cena
    }

    private void GetGameObjectWithTag( string tagName ) {
        // TODO: Pega game object
    }
}

```

Sugestões Exemplo 2 a que acho mais bonita ahahahahah:

```
public sealed class Constants {
    public sealed class SceneName {
        public const string LEVEL_SELECTION = "SelectLvl";
        public const string THREE_VALLEY = "MainLevel";
        public const string MENU = "Menu";
        public const string PAUSE = "Pause";
        public const string OPTIONS = "Options";
    }
    
    public sealed class Tags {
        public const string PLAYER = "Player";
        public const string CAR = "Car";
        public const string MOTO_CROSS = "MotoCross";
        public const string PLAYER_LIFE = "PlayerLife";
        public const string UI_SCORE = "UIScore";
    }
}
//Exemplo de como voce utilizaria == O nome da classe abaixo e so exemplo 
public sealed class MenuSceneController {
    
    private void Update() {
        GotoScene(Constants.SceneName.LEVEL_SELECTION);
        GetGameObjectWithTag(Constants.Tags.PLAYER);
    }
    
    private void GotoScene( string sceneName ) {
        //TODO: Faz ir para outra cena
    }
    
    private void GetGameObjectWithTag( string tagName ) {
        // TODO: Pega game object
    }
}

```
O segundo é que já que estamos falando de constants então elas são imutáveis, ou seja, que ninguém pode alterar, o Getter é desnecessário, normalmente as constante são todas publicas e quem quiser pode acessar já que quando vc declara uma constant do tipo String ela já static por default.